feedback/reporter.go

```diff
@@ -106,6 +112,40 @@ func (r *Reporter) NewIssue(report UserReport) (result *client.CreateGithubIssue
 	return result, nil
 }
 
+// NewIntercomIssue sends node logs, Identity and UserReport to intercom
+func (r *Reporter) NewIntercomIssue(report UserReport) (result *client.CreateIntercomIssueResult, err error) {
+	if err := report.Validate(); err != nil {
+		return nil, err
+	}
+
+	nodeID := r.currentIdentity()
+	location, err := r.locationResolver.DetectLocation()
```

Comment:
As I understand this is only for provider, but maybe `GetOrigin` would still be more correct.